### DISCUSSION TITLE
Sonoma (14.6) Broke My Plash

### DISCUSSION THREAD

**Discussion by fpuhan** [sindresorhus/Plash#166] [Q&A]
In March, 2023, I used Plash, Shortcuts and a little SQL under Ventura to show the names of the images being displayed on my five desktops/spaces.  Those names were found in `~/Library/Application Support/Dock/desktoppicture.db`.  It appears Sonoma no longer uses this sqlite database to store the image file names.  I've poked around plists, done time-relative `find`  commands on `~/Library` and `/System/Library` after changing an image manually, and still cannot locate a new store for this information.

Since Plash is the component that makes it all happen, I wonder if anyone can point me to the new storage location used by Apple to keep desktop image file information?  Is it also sqlite, or will I need to scrape this data another way?  

Thanks in advance.  Plash was the magic "glue" that made it all happen.  I'm delighted with it!

**Comment by CorgFather** [comment]
I found them on Sequoia (15.0) under '/System/Library/Desktop Pictures'

**Reply by fpuhan** [reply]
Those are the wallpaper images that ship with macOS.  I'm looking for the location of the names macOS uses when you set custom folders and images as the desktop wallpaper.
